odoo/tests/common.py

```diff
@@ -2104,7 +2104,7 @@ def _process_fvg(self, model, fvg, level=2):
             order.append(fname)
 
             node_modifiers = {
-                modifier: ([TRUE_LEAF] if domain else [FALSE_LEAF]) if isinstance(domain, int) else normalize_domain(domain)
+                modifier: ([TRUE_LEAF] if domain else [FALSE_LEAF]) if isinstance(domain, int) else normalize_domain(safe_eval(domain, dict(self._env.context), nocopy=True) if isinstance(domain, str) else domain)
```

Comment:
if else if else if else...

could you this PR opportunity to rework this line so that humans can read it?

Reply:
Done